### DISCUSSION TITLE
Long response Tet Does Not Scroll In Pre 4 Release

### DISCUSSION THREAD

**Discussion by mikey60** [toverainc/willow#235] [General]
Long response texts no longer scroll when running the Pre 4 bin file.  These texts did scroll when running  the version of willow that was built by the user (myself).  Is this by design?

**Comment by nikito** [comment]
I believe this is presently by design in the latest code. I think this was
done because the scrolling was causing a big performance hit on one of the
cpu cores that could lead to the device becoming unresponsive.
Thanks and Best Regards,
Nick Bento

**Comment by mikey60** [comment]
@nikito Thank You!

**Comment by stintel** [comment]
See https://github.com/toverainc/willow/pull/253
